### ISSUE TITLE
Command `list databases` does not work via HTTP REST API

### ISSUE THREAD

**Issue by baughmann** [ArcadeData/arcadedb#966]
### Expected behavior
When following the [docs](https://docs.arcadedb.com/#HTTP-ServerCommand), I should be able to `list databases` via the REST API.

Using Postman, making a `POST` request to `http://localhost:2480/api/v1/server` with header `'Content-Type': 'application/json'` and raw body `{"command": "list databases"}` and `Basic Auth` enabled using `root:playwithdata` should result in a HTTP 200 with JSON content `{ "result" : ["root","test"]}`

### Actual behavior
Above request results in HTTP 400 with response body ``{"error": "Server command not valid"}`.

I tested another command, `{"command": "create database test"}` and it executed successfully.

### Steps to reproduce
1. Download the  [docker-compose.yaml](https://github.com/ArcadeData/arcadedb/blob/main/docker-compose.yml) and run `docker compose up`
2. Open postman and create a new `POST` request to `http://localhost:2480/api/v1/server`  with header `'Content-Type': 'application/json'` and raw body `{"command": "list databases"}` 

### Other info
Output of `docker version`
```
Client:
 Cloud integration: v1.0.29
 Version:           20.10.20
 API version:       1.41
 Go version:        go1.18.7
 Git commit:        9fdeb9c
 Built:             Tue Oct 18 18:28:44 2022
 OS/Arch:           windows/amd64
 Context:           default
 Experimental:      true

Server: Docker Desktop 4.13.1 (90346)
 Engine:
  Version:          20.10.20
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.18.7
  Git commit:       03df974
  Built:            Tue Oct 18 18:18:35 2022
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.6.8
  GitCommit:        9cd3357b7fd7218e4aec3eae239db1f68a5a6ec6
 runc:
  Version:          1.1.4
  GitCommit:        v1.1.4-0-g5fd4c4d
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

Am I doing something wrong here or is this an actual bug? Also nice work on this, I'm having a lot of fun using ArcadeDB :)

**Comment by allegrojm** [comment]
Hi, I saw this and tried it in my environment and it works OK, but we are using ArcadeDB Version: v23.3.1-SNAPSHOT. Not sure where/when a fix for this was done. just fyi.

**Comment by allegrojm** [comment]
As an aside, the list databases command doesn't work in Studio,

**Comment by lvca** [comment]
Please use the latest docker image (23.3.1-SNAPSHOT or major if available)

**Comment by lvca** [comment]
@allegrojm just pushed a fix to use the new POST /server command with "list databases" from Studio.

**Comment by baughmann** [comment]
Thanks @allegrojm and @lvca!

I'm home now and only have a Linux environment already on the latest, but tomorrow when I'm back in the office I will pull the latest and re-teat then respond here.

**Comment by baughmann** [comment]
Looks like its fixed in `ARCADEDB 23.3.1-SNAPSHOT`, please close this issue. Thank you all!
